### PR TITLE
Update swagger.yaml and api/version-history.md for DeviceCgrouprules

### PR DESCRIPTION
**Pull request by mlaventure**
Signed-off-by: Kenfe-Mickael Laventure <mickael.laventure@gmail.com>

### CODE REVIEW

**Review by thaJeztah — APPROVED** [review]
LGTM

File: docs/api/version-history.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ keywords: "API, Docker, rcli, REST, documentation"
 
 [Docker Engine API v1.27](https://docs.docker.com/engine/api/v1.27/) documentation
 
+* `GET /containers/creates` now takes a `DeviceCgroupRules` field in `HostConfig` allowing to set custom device cgroup rules for the created container.
```
Comment (justincormack) [inline docs/api/version-history.md]: shouldnt that be `/containers/create`?

**Comment by mlaventure** [comment]
@justincormack correct, updated. Thanks.

**Comment by thaJeztah** [comment]
Ugh, needs a rebase now 😢

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Comment by justincormack** [comment]
@mlaventure needs a rebase!

**Comment by mlaventure** [comment]
rebased

**Comment by vieux** [comment]
LGTM

**Pull request merged**